recipes/gnali/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.1.0rc0.dev2" %}
```

Comment:
The tarball is `0.1.0rc0.dev2`. Is it possible to make this an official release instead?

Reply:
ed3afa6 Updated version and SHA256